app/views/users/new_with_form_tag.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_tag("/users", method: "Post") do %>
```

Comment:
- The default method for [`form_tag`](https://api.rubyonrails.org/) is post and it is written with smaller `p` or as symbol `method: :post`